lib/plugins/aws/package/compile/events/apiGateway/lib/deployment.js

```diff
@@ -13,7 +13,6 @@ module.exports = {
         Type: 'AWS::ApiGateway::Deployment',
         Properties: {
           RestApiId: { Ref: this.apiGatewayRestApiLogicalId },
-          StageName: this.options.stage,
```

Comment:
Isn't this removal introducing a breaking change for developers relying on this information?